gaffer-as-a-service/gaas-rest/src/test/java/uk/gov/gchq/gaffer/gaas/handlers/TestAppender.java

```diff
@@ -0,0 +1 @@
+/*
 * Copyright 2021 Crown Copyright
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

package uk.gov.gchq.gaffer.gaas.handlers;

import org.apache.log4j.AppenderSkeleton;
import org.apache.log4j.spi.LoggingEvent;
import java.util.ArrayList;
import java.util.List;

class TestAppender extends AppenderSkeleton {

    private final List<LoggingEvent> log = new ArrayList<LoggingEvent>();

    @Override
    public boolean requiresLayout() {
        return false;
    }

    @Override
    protected void append(final LoggingEvent loggingEvent) {
        log.add(loggingEvent);
    }

    @Override
    public void close() {

    }

    public List<LoggingEvent> getLog() {
        return new ArrayList<LoggingEvent>(log);
    }

}

```

Comment:
Incorrect line endings in this file.

Reply:
@GCHQDeveloper314 , @t92549,
 
Removed the incorrect line from the file

Reply:
Line endings are still incorrect, I am seeing `^M` characters.

Reply:
@GCHQDeveloper314 

That is correct the line endigs are still there. There is no issue with class when we run checkstyle. It seems that Github formats differently.

Reply:
I have fixed it for you. The file had carriage return line endings, but they need to be line feed to work correctly in GitHub (all files use line feed), [see here for more info on this phenomenon](https://en.wikipedia.org/wiki/Newline).